### PR TITLE
Refs #26601 -- Used new-style middlewares in documentation.

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

File: docs/topics/http/middleware.txt [hunk 1]
```diff
@@ -292,8 +292,7 @@ Upgrading pre-Django 1.10-style middleware
 
 Django provides ``django.utils.deprecation.MiddlewareMixin`` to ease creating
 middleware classes that are compatible with both :setting:`MIDDLEWARE` and the
-old ``MIDDLEWARE_CLASSES``. All middleware classes included with Django
-are compatible with both settings.
```
Comment (felixxm) [inline docs/topics/http/middleware.txt]: IMO this sentence is still true. I don't think that we need to remove it due to this documentation change.

Comment (claudep) [reply docs/topics/http/middleware.txt]: True, this was more related to an other PR I was preparing separately.

**Comment by claudep** [comment]
Grrr, this CI server is definitely dumb!

**Comment by felixxm** [comment]
@claudep Thanks :+1:

**Pull request merged**